app/views/devise/shared/_password_strength.html.slim

```diff
@@ -7,6 +7,6 @@
       .pw-bar
     .h5
       span.h6 = t('instructions.password.strength.intro')
-      span#pw-strength-txt.bold = '...'
+      span#pw-strength-txt.bold*{ 'data-forbidden-passwords'=>@forbidden_passwords } = '...'
```

Comment:
Looks like slim lint is made that you didn't put spaces around your hash rocket here